docs/Style.thy

```diff
@@ -87,11 +88,17 @@ text \<open>
   the closing bracket on the same line as the ending text to not waste too much vertical space.
   Indent text by 2 inside the @{command text} area. This achieves visual separation.\<close>
 
+
 section \<open>Indentation\<close>
 
 text \<open>
-  Isabelle code is much easier to maintain when indented consistently. In apply style proofs we
-  indent by 2 spaces, and add an additional space for every additional subgoal.
+  Isabelle code is much easier to maintain when indented consistently.
+  When in doubt, and not constrained by vertically aligning items or subgoal-count offsets, use
+  2 spaces when indenting something with respect to its container (see ``General layout`` in
+  ``Other`` section).
+
+  In apply style proofs we indent by 2 spaces, and add an additional space for every additional
```

Comment:
I know this is how it was phrased in the previous version, but would it be a bit more clear to say after what's here something like "for example, a command which applies when there are 3 subgoals should be indented by 4 spaces".

Reply:
Sure, I can add that while we're here.

Reply:
Added, let me know if it's clear now.

Reply:
Yep, thanks.